frontend/model/contracts/chatroom.js

```diff
@@ -396,7 +397,7 @@ sbp('chelonia/defineContract', {
     },
     'gi.contracts/chatroom/editMessage': {
       validate: actionRequireInnerSignature(objectOf({
-        hash: string,
+        hash: stringMax(MAX_HASH_LEN, 'hash'),
```

Comment:
Just below this is `text: string`, can you modify that to be `text: stringMax(MAX_MESSAGE_LEN, 'text')`?

Where `MAX_MESSAGE_LEN` is a new constant of `20000`?

This will also require updating `addMessage` and the UI to add the same hard limit to both the edit field and the submit message field in the UI.

Reply:
@taoeffect can this be worked on as a separate issue?

Reply:
Yeah 👍

Reply:
Although we can still add that limit here

Reply:
> Although we can still add that limit here, and later do the UI

Sounds good.